_layouts/home.html

```diff
@@ -81,10 +81,10 @@ <h3 class="line-height-medium">Make your organizing more local and participatory
         </div>
       </div>
 
-      <div class="col-md-7 pl-0 pr-0 pl-md-1 pr-md-1">
-        <h5 class="d-block d-md-none mt-3 pr-1 pl-1">Read Our Case Studies:</h5>
+      <div class="col-md-7 ps-0 pe-0 ps-md-1 pe-md-1">
+        <h5 class="d-block d-md-none mt-3 pe-1 ps-1">Read Our Case Studies:</h5>
         <div class="d-block d-md-none">
-          <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+          <div id="caseStudiesCarousel" class="carousel slide" data-bs-ride="carousel">
```

Comment:
Should we use the same `carousel-dark` style here for consistency, or the style doesn't look right for this carousel context?

Reply:
Good question! This one is on a white background, so `carousel-dark` is not needed.